code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -195,21 +195,9 @@
 	desc = "A lovely little box filled with soft, cute plushies, perfect for calming down people who have just suffered a traumatic event. Legend has it there's a special part of hell \
 	for Medical Officers who just take the box for themselves."
 
-	/// the plushies that aren't of things trying to kill you
-	var/list/static/approved_by_corporate = list(
-		/obj/item/toy/plush/carpplushie, // well, maybe they can be something that tries to kill you a little bit
-		/obj/item/toy/plush/slimeplushie,
-		/obj/item/toy/plush/lizard_plushie,
-		/obj/item/toy/plush/snakeplushie,
-		/obj/item/toy/plush/plasmamanplushie,
-		/obj/item/toy/plush/beeplushie,
-		/obj/item/toy/plush/moth,
-		/obj/item/toy/plush/pkplush,
-	)
-
 /obj/item/storage/box/hug/plushes/PopulateContents()
 	for(var/i in 1 to 7)
-		var/plush_path = pick(approved_by_corporate)
+		var/plush_path = pick(/obj/effect/spawner/random/entertainment/plushie)
```

Comment:
I don't think this'll work

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/70091/commits/42eb0fed4004ef6c8c6c557a2093a9dc5c75dd96